### ISSUE TITLE
Support for inserting a line after a matching line is found

### ISSUE THREAD

**Issue by vorezal** [sous-chefs/line#79]
It would be great if this cookbook had the ability to search for a matching line of text and then insert another line after it. It would certainly help keep things organized in config files and the like when templates are not a good option.

I'm going to be away for a bit, but if no one tackles it before I have time I can certainly give it a go myself and submit a PR. I think it should be similar to the replace_or_add functionality (insert_or_add perhaps?), but I can think of a number of edge cases that would need addressing.

**Comment by MarkGibbons** [comment]
Current syntax for this request in #101
````
filters = Line::Filter.new
filter_lines '/some/file' do
   filter filters.method(:after)
   filter_args [/pattern/, 'insert_line']
end
````

**Comment by vorezal** [comment]
That looks excellent to me. Way above and beyond what I had been imagining no less. I'm interested to look through the code and see how it all works. Thank you for addressing my request!

**Comment by MarkGibbons** [comment]
Fixed by #121
